ui/decredmaterial/container.go

```diff
@@ -0,0 +1,83 @@
+package decredmaterial
+
+import (
+	"gioui.org/layout"
+	"gioui.org/unit"
+)
+
+type Container struct {
+	container *layout.List
+	scrollbar *Scrollbar
+
+	contentHeight float32
+}
+
+const (
+	scrollbarWidth = 10
+)
+
+func (t *Theme) Container() *Container {
+	return &Container{
+		container: &layout.List{
+			Axis: layout.Vertical,
+		},
+		scrollbar: t.Scrollbar(),
+	}
+}
+
+func (c *Container) calculateContentHeight(gtx layout.Context, w []func(gtx C) D) {
```

Comment:
kindly add comments here and to other methods whose function can't be deduced at first glance.